src/Compiler/Checking/QuotationTranslator.fs

```diff
@@ -708,7 +709,7 @@ and private ConvExprCore cenv (env : QuotationTranslationEnv) (expr: Expr) : QP.
             wfail(Error(FSComp.SR.crefQuotationsCantRequireByref(), m))
 
         | TOp.TraitCall traitInfo, _, args ->
-            let g = g
+            //let g = g
```

Comment:
I guess just remove it?

Reply:
It must be someone's important debug-location...